operator/controller/pom.xml

```diff
@@ -116,6 +131,49 @@
           </execution>
         </executions>
       </plugin>
+
     </plugins>
+
   </build>
+
+  <profiles>
+    <profile>
+      <id>prepare-operator-tests</id>
```

Comment:
I'm not sure if I'm following here, currently:
- we use resources exclusively from code in the testsuite

with this change:
- we use `make` in the Maven build to generate resources already available in Java

What would be a better separation of concerns?
Can we use exclusively Java?
Alternatively, we should trigger the `mvn` commands from the `Makefile` where concatenating those steps feels more natural.
wdyt?

Reply:
We're not using resources generated by QOSDK because we're not using them when deploying in production. We should match the test environment as close as possible to the run environment.

> Alternatively, we should trigger the mvn commands from the Makefile where concatenating those steps feels more natural.

We could, that would just require users to use make to run the tests (or concatenate it manually). I'm fine with the current approach at least for now.

Reply:
Done.